### ISSUE TITLE
Extra standart ribbon in plot_erp

### ISSUE THREAD

**Issue by vladdez** [unfoldtoolbox/UnfoldMakie.jl#238]
![image](https://github.com/user-attachments/assets/cae0087c-024a-4444-a8ec-1366b2575be7)

**Comment by behinger** [comment]
possibly related to:
https://github.com/unfoldtoolbox/UnfoldMakie.jl/issues/55

my guess is that some `group`/`col`/`row` variables are not correctly applied to the ribbon

**Comment by vladdez** [comment]
It seem like they have never been applied at all

**Comment by behinger** [comment]
possibly easy fix then :)

**Comment by behinger** [comment]
but be aware that a `mapping*(visual+ribbon)` would apply them to both, so that could be a good solution; not sure how it is implemented right now

**Comment by vladdez** [comment]
currently it is implemented like this:

```
    # check if stderror values exist and create new columns with high and low band
    if "stderror" ∈ names(plot_data) && stderror
        plot_data.stderror = plot_data.stderror .|> a -> isnothing(a) ? 0.0 : a
        plot_data[!, :se_low] = plot_data[:, config.mapping.y] .- plot_data.stderror
        plot_data[!, :se_high] = plot_data[:, config.mapping.y] .+ plot_data.stderror
    end
   ...
    if stderror
        m_se = AlgebraOfGraphics.mapping(config.mapping.x, :se_low, :se_high)
        @debug m_se
        basic = basic + visual(Band, alpha = 0.5) * m_se
    end 
```

**Comment by vladdez** [comment]
fixed in #237
